card/config/locales/de.yml

```diff
@@ -174,6 +185,25 @@ de:
             #description: Beschreibung
             comment_style_group: STYLE GROUP
             search_opt_names_ptr: option names for pointer
```

Comment:
I hesitated to translate "pointer". "Zeiger" probably makes only sense to coders but German coders will understand "pointer". It's maybe even more familiar than "Zeiger". I'm still a fan of the idea to change the name to something like "list".

Reply:
For the moment, I think it's ok not to translate things like cardtypes, because we can't avoid having users see the English anyway.  (Since we don't yet handle translating card names).  I was more thinking of the "option names for" part.

I think I'm coming around to "List".